### ISSUE TITLE
Include nightly versions

### ISSUE THREAD

**Issue by codethief** [asdf-community/asdf-zig#4]
The Bash script retrieves https://ziglang.org/download/index.json, which lists the current v11.0-dev.* nightly versions. However, when I run `asdf list all zig` I only see the stable versions (0.1.1 through 0.10.1).

Could this be changed to also include nightly versions?

Unfortunately, I can't really tell where in the script the filtering happens, otherwise I'd change it myself.

**Comment by jhcl** [comment]
When I run the same command (`asdf list all zig`) I see 0.2.0 through 0.10.1 and master.  
`asdf install zig master` gives me version 0.11.0-dev.1884+2641feb9b.

**Comment by jhcl** [comment]
- I am not the maintainer, just a passer-by. It would be nice to have even more versioning options for such a fast moving target but I don't think asdf tracks github commits, just github labels (see comment in https://github.com/cheetah/asdf-zig/issues/5).
- As far as I know master is the nightly build. 
- When i do a `asdf plugin list-all | grep zig` I get a reference to https://github.com/cheetah/asdf-zig.git. So I'm not sure we are discussing this in the correct repository. Perhaps the repo owner can enlighten us.

**Comment by tobyhinloopen** [comment]
yeah i wish `master` was included in `list-all`. Eventually I found out you can just install `master`

**Comment by chenrui333** [comment]
Any luck with this thread?

**Comment by jiacai2050** [comment]
This should be fixed in latest master, feel free to open new issue if you have more questions.

**Comment by ice-chillios** [comment]
@jiacai2050 I think it's still filtering master there: https://github.com/asdf-community/asdf-zig/blob/master/lib/utils.py#L61

<img width="194" height="360" alt="Image" src="https://github.com/user-attachments/assets/116c5339-e279-4d0d-921b-7c9e77fa0f35" />

**Comment by jiacai2050** [comment]
IMO there is no such `master` version, `master` is a just a reference to the latest dev commit, so `list all` will not show it.

However you can still install it with `asdf zig install master` since there is no version check when install.
